Adds Spire, a semi-planet map

Dogbone10 opened the pull request:
<!-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->

## About the PR
Adds a map called Spire, made by me, PenguinCodest and Jack_the_dragon with help from other members of the harmony community

## Why / Balance
<!-- Discuss how this would affect game balance or explain why it was changed. Link any relevant discussions or issues. -->
Playing on Glacier from Delta-V/EE servers has made me realize how much opportunity for gameplay a planetary atmosphere adds. I want one for us too!

## Technical details
<!-- Summary of code changes for easier review. -->
Parallax and atmos currently not shown in images, image may be partially outdated/not the current build.
Map can be loaded manually with mapping or loadmap commands, not at roundstart. Needs to be amended.

## Media
![Spire-0](https://github.com/user-attachments/assets/f137adbb-f085-426a-b8ae-65a66222d42c)

## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an in-game showcase.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

## Breaking changes
Map does **not load** at the moment, and does not render either. Needs to be fixed, but I am at my whits end trying to patch it up. Putting it up here so I can track my progress better.
I also need to pull out a lot of extra changes I was messing around with in my dev environment.
It shouldn't change anything else besides adding a map and a prototype for said map.
If anyone has any idea what is wrong with it, please help. I am at my whits end with it flip-flopping between working and not

**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->
<!--
:cl:
- add: Spire has been added. This map has a cold, hostile outside atmosphere, so make sure to bundle up and keep an air tank on hand!
- remove: 
- tweak: 
- fix: 
-->

spanky-spanky commented on Resources/Prototypes/_Harmony/Maps/spire.yml:
```diff
@@ -0,0 +1,63 @@
+  minPlayers: 0
+  maxPlayers: 25
+  stations:
+    Mira:
```
This needs to match the name in the BecomeStation component on your map, probably `Spire`

spanky-spanky commented on Resources/Prototypes/_Harmony/Maps/spire.yml:
```diff
@@ -0,0 +1,63 @@
+            Detective: [ 1, 1 ]
+            SecurityOfficer: [ 4, 5 ]
+            SecurityCadet: [ 3, 3 ]
+            PermaPrisoner: [ 2, 0 ] # HARMONY
```
You don’t need the Harmony comment since this is in our namespace.

jack-thedragon95 commented:
downloaded it and tried to run it something is FUCKED in this maybe get rid of some of the custom shit?

Dogbone10 commented:
That right there is the FINAL update needed before it is a playable map.
Me, PenguinCodest and Jack the Dragon redid the entirety of medical, moved the bar, added genpop and redid perma, scooched Detective over, widened lawyer, expanded maints, double-checked a lot of external airlocks, got the power grid hooked up, threw in MORE penguins, fancied up engineering front and chemistry, made the salvager cabin more cabin-y, added an external jetpack, moved the mime/clown down and removed their musician buddy's room, gave musician a slave-cage amongst the nitrogen bar, forgot to re-add Pochito (whoops!), moved the HoP to make EVA more break-in-able, put HoP and Ian's beds together, made a murder-alley near chapel, removed trash.
Uh.. Thats pretty much everything!
I can not thank these two enough for all their help, especially Penguin who has been my co-mapper and voice of reason throughout this, and Jack for being a huge designer the food-side of service. Either before this publishes or after he'll add some Shoal banners, and we'll be adding those to the bar.
Maybe we'll also finally add that maints-bar as a cabin in the ice we wanted to. Awaiting criticism!

spanky-spanky commented:
Please keep art files out of this in order to simply review process. Add this banner in another PR and map it in a future update.

spanky-spanky commented:
The bugs got in your PR somehow
